example/issue_comment.go

```diff
@@ -65,16 +65,21 @@ func (h *PRCommentHandler) Handle(ctx context.Context, eventType, deliveryID str
 
 	repoOwner := repo.GetOwner().GetLogin()
 	repoName := repo.GetName()
-	author := event.GetComment().GetUser()
+	author := event.GetComment().GetUser().GetLogin()
 	body := event.GetComment().GetBody()
```

Comment:
Without this change, the logs (and the comment) are a serialization of the full User object, which is big and ugly